src/test/resources/pacts/publicapi-connector-get-payment-refunds.json

```diff
@@ -0,0 +1,210 @@
+      },
+      "response": {
+        "status": 200,
+        "headers": {
+          "Content-Type": "application/json"
+        },
+        "body": {
+          "payment_id": "charge8133029783750222",
+          "_links": {
+            "self": {
+              "href": "https://connector/v1/api/accounts/123456/charges/charge8133029783750222/refunds"
+            },
+            "payment": {
+              "href": "https://connector/v1/api/accounts/123456/charges/charge8133029783750222"
+            }
+          },
+          "_embedded": {
+            "refunds": [
+              {
+                "amount": 1,
+                "created_date": "2016-01-25T13:23:55.000Z",
+                "refund_id": "di0qnu9ucdo7aslhatci6h90jk",
+                "user_external_id": "d77458fb878b4aba88b8164368be3d16",
+                "status": "success",
+                "_links": {
+                  "self": {
+                    "href": "https://app.com/v1/api/accounts/123456/charges/charge8133029783750222/refunds/di0qnu9ucdo7aslhatci6h90jk"
+                  },
+                  "payment": {
+                    "href": "https://app.com/v1/api/accounts/123456/charges/charge8133029783750222"
+                  }
+                }
+              },
+              {
+                "amount": 1,
+                "created_date": "2016-01-25T16:23:55.000Z",
+                "refund_id": "m16ufgc3t23l766ljhv9eicsn5",
+                "user_external_id": "d77458fb878b4aba88b8164368be3d16",
+                "status": "error",
+                "_links": {
+                  "self": {
+                    "href": "https://app.com/v1/api/accounts/123456/charges/charge8133029783750222/refunds/m16ufgc3t23l766ljhv9eicsn5"
+                  },
+                  "payment": {
+                    "href": "https://app.com/v1/api/accounts/123456/charges/charge8133029783750222"
+                  }
+                }
+              }
+            ]
+          }
+        },
+        "matchingRules": {
+          "body": {
+            "$.payment_id": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$._links.self.href": {
+              "matchers": [
+                {
+                  "regex": "http.*://.*/v1/api/accounts/123456/charges/charge8133029783750222/refunds"
+                }
+              ]
+            },
+            "$._links.payment.href": {
+              "matchers": [
+                {
+                  "regex": "http.*://.*/v1/api/accounts/123456/charges/charge8133029783750222"
+                }
+              ]
+            },
+            "$._embedded.refunds[0].amount": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$._embedded.refunds[0].created_date": {
+              "matchers": [
+                {
+                  "match": "value"
```

Comment:
Should it be value, type or datetime pattern in the case of `created_date`?

Reply:
I think value is fine. Not exactly sure why we have patterns in some places, but one reason I can think of is createdDate set to ZonedDateTime.now() in states which makes it hard to check for value